tests/test-utils.js

```diff
@@ -0,0 +1,29 @@
+export async function fundAccount(
+  web3,
+  sender,
+  accounts,
+  maxFeePerGas,
+  maxPriorityFeePerGas,
+  nonce,
+  value
+) {
+  const tx = {
+    from: sender.address,
+    to: accounts[0],
+    value,
+    nonce,
+    gasLimit: 22000,
+    maxFeePerGas,
+    maxPriorityFeePerGas
+  }
+  await web3.eth.sendTransaction(tx)
+}
+
+export function santizeIterations(iter) {
```

Comment:
typo in `santizeIterations` > `sanitizeIterations`

Reply:
Fixed: [b4b3b72](https://github.com/maticnetwork/matic-cli/pull/169/commits/b4b3b72cf69dec4e36007d33bcc1987fd189dc2f)